docs/API.md

```diff
@@ -708,10 +708,10 @@ __Parameters__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`bucketName`  | _string_  |name of the bucket.   |
-|`objectName` | _string_  |name of the object.   |
-|`expiry` | _time.Duration_  |expiry in seconds.   |
-|`reqParams` | _url.Values_  |additional response header overrides supports _response-expires_, _response-content-type_, _response-cache-control_, _response-content-disposition_.  |
+|`bucketName`  | _string_  |Name of the bucket   |
+|`objectName` | _string_  |Name of the object   |
+|`expiry` | _time.Duration_  |Expiry of presigned URL in seconds   |
+|`reqParams` | _url.Values_  |Additional response header overrides supports _response-expires_, _response-content-type_, _response-cache-control_, _response-content-disposition_.  |
```

Comment:
Might be missing a comma here "Additional response header overrides, supports ... "